ravi.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_FIREBASE_API_KEY=AIzaSyAbqqXtHNIuNrsamkCxRk9sOuMO-ZWDiEk
+REACT_APP_FIREBASE_AUTH_DOMAIN=codeforcauseorg.firebaseapp.com
+REACT_APP_APP_VERSION=2.0.0
+REACT_APP_API_BASE_URL=/api
+REACT_APP_ENABLE_REDUX_LOGGER=false
```

Comment:
It is not needed, env.example is already there.

I think you added it by mistake.